libs/tendermint/blockchain/v0/reactor.go

```diff
@@ -202,6 +203,20 @@ func (bcR *BlockchainReactor) respondToPeer(msg *bcBlockRequestMessage,
 
 // Receive implements Reactor by handling 4 types of messages (look below).
 func (bcR *BlockchainReactor) Receive(chID byte, src p2p.Peer, msgBytes []byte) {
+	if cfg.DynamicConfig.GetEnableConsensusIPWhitelist() {
+		okIP := false
+		for _, ip := range cfg.DynamicConfig.GetConsensusIPWhitelist() {
```

Comment:
use map to check contains okip

Reply:
and v1 v2 aslo need check okip

Reply:
Rececive may recv request message like 'bcBlockRequestMessage,.bcStatusRequestMessage' . these message can not block